powderday/nebular_emission/cloudy_model.py

```diff
@@ -184,7 +185,7 @@ def get_nebular(spec_lambda, sspi, nh, logq, radius, logu, logz, logq_1, Dust=Fa
     model_name = filename.split(".")[0]
 
     logging.info("Writing CLOUDY input file")
-    dir_= os.getcwd() + "/powderday/nebular_emission"
+    dir_= cfg.par.pd_source_dir + "/powderday/nebular_emission"
```

Comment:
i think this won't work becuase the code needs to make the directory if it doesn't exist (otherwise it'll crash).  how about something like:

```    dir_ = cfg.par.PD_output_dir+"/nebular_emission"
    if not os.path.exists(dir_):
        os.makedirs(dir_)
    dir_tempfiles_ = dir_+"/temp_files"
    if not os.path.exists(dir_tempfiles_):
        os.makedirs(dir_tempfiles)```

Reply:
The directory already exists. It is how pd is shipped.

Reply:
just kiddin gmy mistake - i didn't realize you were using `pd_source_dir`. that's smart